tutorials/algorithms/04_vqd.ipynb

```diff
@@ -163,15 +163,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 28,
    "metadata": {},
    "outputs": [],
    "source": [
     "from qiskit.algorithms.eigensolvers import VQD\n",
+    "from qiskit.algorithms.eigensolvers import NumPyEigensolver\n",
+    "\n",
     "\n",
     "vqd = VQD(estimator, fidelity, ansatz, optimizer, k=k, betas=betas, callback=callback)\n",
     "result = vqd.compute_eigenvalues(operator = H2_op)\n",
-    "vqd_values = result.optimal_values"
+    "vqd_temp = result.eigenvalues\n",
+    "vqd_values = vqd_temp"
```

Comment:
There is no need for this to be changed at all, It was correct as it was before. Yes it may have this other field too but then,as you needed to do below, since its complex, you need to deal with that.

Reply:
My apologies for the wasted time, I attached the wrong .ipynb file initially. I will make remove these changes.